### PR TITLE
Migrate MotorControl to be compatible with SageResearch 4.0

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
@dephillipsmichael

### CODE REVIEW

File: MCTSampleApp/MCTSampleApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -367,9 +367,9 @@
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 3;
+				CURRENT_PROJECT_VERSION = 0;
```
Comment (Erin-Mounts) [inline MCTSampleApp/MCTSampleApp.xcodeproj/project.pbxproj:370]: Is this intentional (resetting the project version to 0)?

File: MCTSampleApp/MCTSampleApp/Info.plist [hunk 2]
```diff
@@ -15,9 +15,9 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.2</string>
+	<string>3.0</string>
 	<key>CFBundleVersion</key>
-	<string>3</string>
+	<string>0</string>
```
Comment (Erin-Mounts) [inline MCTSampleApp/MCTSampleApp/Info.plist:20]: Intentional? Should this "match" the short version string?

File: MCTSampleApp/MCTSampleAppTests/Info.plist [hunk 3]
```diff
@@ -15,8 +15,8 @@
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.2</string>
+	<string>3.0</string>
 	<key>CFBundleVersion</key>
-	<string>3</string>
+	<string>0</string>
```
Comment (Erin-Mounts) [inline MCTSampleApp/MCTSampleAppTests/Info.plist:20]: Intentional?

File: MotorControl/MotorControl.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -907,7 +907,7 @@
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 3;
+				CURRENT_PROJECT_VERSION = 0;
```
Comment (Erin-Mounts) [inline MotorControl/MotorControl.xcodeproj/project.pbxproj:910]: Intentional?

File: MotorControl/MotorControl/Info.plist [hunk 5]
```diff
@@ -15,9 +15,9 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.2</string>
+	<string>3.0</string>
 	<key>CFBundleVersion</key>
-	<string>3</string>
+	<string>0</string>
```
Comment (Erin-Mounts) [inline MotorControl/MotorControl/Info.plist:20]: Intentional?

File: MotorControl/MotorControlTests/Info.plist [hunk 6]
```diff
@@ -15,8 +15,8 @@
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.2</string>
+	<string>3.0</string>
 	<key>CFBundleVersion</key>
-	<string>3</string>
+	<string>0</string>
```
Comment (Erin-Mounts) [inline MotorControl/MotorControlTests/Info.plist:20]: Intentional?

**Review by Erin-Mounts — APPROVED** [review]
Question(s) about resetting versions to 0, otherwise looks good

File: MCTSampleApp/MCTSampleApp.xcodeproj/project.pbxproj (hunk 1 above)
Comment (syoung-smallwisdom) [reply MCTSampleApp/MCTSampleApp.xcodeproj/project.pbxproj:370]: This is the build. The project version is the marketing version (3.0) and the build (0) which then are combined for a semantic version of 3.0.0

**Pull request merged**